scripts/mem0ai/mem0_mcp_server.py

```
@@ -58,9 +56,7 @@ def do_GET(self):
                 self.wfile.write(json.dumps({"data": memory}).encode())
             except Exception as e:
                 self._set_response(404)
-                self.wfile.write(
-                    json.dumps({"error": f"Error getting memory: {str(e)}"}).encode()
-                )
+                self.wfile.write(json.dumps({"error": f"Error getting memory: {str(e)}"}).encode())
```

Ruff (0.11.9)

Use explicit conversion flag

Replace with conversion flag

(RUF010)